MIGRATION_GUIDE.md

```diff
@@ -78,6 +78,14 @@ To easily handle three-value logic (true, false, unknown) in provider's configs,
 #### *(note)* `resource_monitor` validation and diff suppression
 `resource_monitor` is an identifier and handling logic may be still slightly changed as part of https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/ROADMAP.md#identifiers-rework. It should be handled automatically (without needed manual actions on user side), though, but it is not guaranteed.
 
+#### *(behavior change)* snowflake_databases datasource
```

Comment:
I already wrote migration guide for snowflake_databases (around line 180), could you merge them?

Reply:
It was supposed to be for warehouses (I will change the name in the follow-up PR).